ui/pages/settings/networks-tab/custom-content-search/custom-content-search.stories.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
```

Comment:
Should we expand this story? It seems like this file is incomplete (`onSearch` and `networksList` are being provided with no values).

Reply:
Might be work adding a new story to fully utilize your test data updates @VSaric. Create the following file `ui/pages/settings/networks-tab/networks-list/network-list.stories.js`

Add this:
```
import React from 'react';
import testData from '../../../../../.storybook/test-data';
import NetworksList from './networks-list';

export default {
  title: 'Pages/Settings/NetworksTab/NetworksList',
  id: __filename,
  argTypes: {
    networkDefaultedToProvider: {
      control: 'boolean',
    },
    networkIsSelected: {
      control: 'boolean',
    },
    networksToRender: {
      control: 'array',
    },
  },
  args: {
    networkDefaultedToProvider: false,
    networkIsSelected: false,
    networksToRender: testData.networkList,
  },
};

export const NetworksListComponent = (args) => {
  return <NetworksList {...args} />;
};

```

Will create the story

<img width="1440" alt="Screen Shot 2022-03-29 at 2 35 47 PM" src="https://user-images.githubusercontent.com/8112138/160711457-9f349bbd-98a5-4e09-9124-82d5d9826fcb.png">